unix/povray.1

```diff
@@ -1,49 +1,41 @@
-.TH POV-Ray 1 "August 2013" "POV\-Team" "Version 3.7" \" -*- nroff -*-
+.TH POV-Ray 1 "December 2016" "POV\-Team" "Version 3.71" \" -*- nroff -*-
```

Comment:
Should probably still be `3.7` - if the version number was intended to refer specifically to 3.7.0, as opposed to the "3.7 generation" as a whole, I would expect it to have read `3.7.0` all along.